src/core/src/OmniTileset.cpp

```diff
@@ -504,6 +505,30 @@ void OmniTileset::reload() {
             }
         }
     }
+
+    const auto& allOmniPolygonRasterOverlays = _pContext->getAssetRegistry().getPolygonRasterOverlays();
+    const auto& boundNativeRasterOverlays =
+        _pTileset->getOverlays().getOverlays(); // get RasterOverlays from RasterOverlayCollection
+    for (const auto& pOmniPolygonRasterOverlay : allOmniPolygonRasterOverlays) {
+        const auto& pNativeOverlay = pOmniPolygonRasterOverlay->getRasterOverlay();
+
+        auto matchIterator = std::find_if(
+            boundNativeRasterOverlays.begin(),
+            boundNativeRasterOverlays.end(),
+            [&pNativeOverlay](const auto& possibleMatch) { return possibleMatch.get() == pNativeOverlay; });
+
+        if (matchIterator == boundNativeRasterOverlays.end()) {
+            continue;
+        }
+
+        const auto exclude = pOmniPolygonRasterOverlay->getExcludeSelectedTiles();
+        if (exclude) {
+            const auto excluder = pOmniPolygonRasterOverlay->getExcluder();
+            if (excluder) {
+                _pTileset->getOptions().excluders.push_back(excluder);
```

Comment:
Do excluders need to be manually removed in `destroyNativeTileset`? I needed to do that for raster overlays to avoid a crash. I haven't had a chance to debug yet and see if it's needed.

Reply:
Seems like it's working fine without.

Reply:
Yeah, I don't think it's needed. I saw that removal code in Unreal, but figured since they're a `std::vector<std::shared_ptr>` then they should destroy because in Omniverse the raster overlays don't talk to the tilesets.